src/main/java/org/mifos/processor/bulk/camel/routes/AccountLookupCallbackRoute.java

```diff
@@ -0,0 +1,111 @@
+package org.mifos.processor.bulk.camel.routes;
+
+import static org.mifos.processor.bulk.camel.config.CamelProperties.LOCAL_FILE_PATH;
+import static org.mifos.processor.bulk.camel.config.CamelProperties.OVERRIDE_HEADER;
+import static org.mifos.processor.bulk.camel.config.CamelProperties.RESULT_TRANSACTION_LIST;
+import static org.mifos.processor.bulk.camel.config.CamelProperties.SERVER_FILE_NAME;
+import static org.mifos.processor.bulk.camel.config.CamelProperties.TRANSACTION_LIST;
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.PARTY_LOOKUP_SUCCESSFUL_TRANSACTION_AMOUNT;
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.PARTY_LOOKUP_SUCCESSFUL_TRANSACTION_COUNT;
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.RESULT_FILE;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.mifos.processor.bulk.schema.BatchAccountLookupResponseDTO;
+import org.mifos.processor.bulk.schema.BeneficiaryDTO;
+import org.mifos.processor.bulk.schema.Transaction;
+import org.mifos.processor.bulk.schema.TransactionResult;
+import org.mifos.processor.bulk.utility.Utils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AccountLookupCallbackRoute extends BaseRouteBuilder {
+
+    @Autowired
+    private ZeebeClient zeebeClient;
+    private Integer totalApprovedAmount;
```

Comment:
please do not create instance variables. this can create accidental side effects